config/routes.rb

```diff
@@ -1,7 +1,17 @@
 Rails.application.routes.draw do
-  root "users#index"
+  #root "users#index"
  
-  resources :users , only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+  get 'likes/new'
+  get 'likes/create'
+  get 'new/create'
+  get 'comments/new'
+  get 'comments/create'
```

Comment:
- [ ] All these new lines of code are like  repetitions of what you did in line 10 -13. Kindly remove it. You are supposed to nest the route in order to achieve the project goal. ✔️ 

- [ ]  Please remove the commented code in line 2. 🙏